src/status_im/contexts/communities/actions/accounts_selection/style.cljs

```diff
@@ -16,4 +17,5 @@
   []
   {:padding-top        12
    :padding-horizontal screen-horizontal-padding
-   :background-color   (colors/theme-colors colors/white colors/neutral-95)})
+   :background-color   (colors/theme-colors colors/white colors/neutral-95)
+   :padding-bottom     (+ (safe-area/get-bottom) 12)})
```

Comment:
What does iOS look like with these changes? Will adding safe-area plus the bottom-padding be consistent for both platforms?

Reply:
yes it considers the notch area (if the device has) + 12px bottom space according to the design